configs/httpd/dist-git/lookaside-upload.conf.example

```diff
@@ -15,93 +15,54 @@ SSLCryptoDevice builtin
 # SSL host
 <VirtualHost _default_:443>
     # This alias must come before the /repo/ one to avoid being overridden.
-    ScriptAlias /repo/pkgs/upload.cgi /var/lib/dist-git/web/upload.cgi
+    ScriptAlias /repo/pkgs/upload.cgi /srv/web/upload.cgi
 
-    Alias /repo/ /var/lib/dist-git/cache/lookaside/
-
-    #ServerName pkgs.fedoraproject.org
-    #ServerAdmin webmaster@fedoraproject.org
+    Alias /repo/ /srv/cache/lookaside/
+    ServerName pkgs.fedoraproject.org
+    ServerAdmin webmaster@fedoraproject.org
 
     SSLEngine on
 
-    # Server Certificate:
-    # Point SSLCertificateFile at a PEM encoded certificate.  If
-    # the certificate is encrypted, then you will be prompted for a
-    # pass phrase.  Note that a kill -HUP will prompt again.  A new
-    # certificate can be generated using the genkey(1) command.
-    SSLCertificateFile /etc/pki/tls/certs/localhost.crt
-
-    # Server Private Key:
-    # If the key is not combined with the certificate, use this
-    # directive to point at the key file.  Keep in mind that if
-    # you've both a RSA and a DSA private key you can configure
-    # both in parallel (to also allow the use of DSA ciphers, etc.)
-    SSLCertificateKeyFile /etc/pki/tls/private/localhost.key
-
-    # Certificate Authority (CA):
-    # Set the CA certificate verification path where to find CA
-    # certificates for client authentication or alternatively one
-    # huge file containing all of them (file must be PEM encoded)
-    SSLCACertificateFile /etc/pki/tls/certs/ca-bundle.crt
-
-    # Certificate Revocation Lists (CRL):
-    # Such a file is simply the concatenation of the various
-    # PEM-encoded CRL files, in order of preference.
-    # These are used to revoke the client certificate on 
-    # Client Authentication.
+    SSLCertificateFile    conf/pkgs.fedoraproject.org_key_and_cert.pem
+    SSLCertificateKeyFile conf/pkgs.fedoraproject.org_key_and_cert.pem
+    SSLCACertificateFile  conf/cacert.pem
     SSLCARevocationFile  /etc/pki/tls/crl.pem
 
-    SSLCipherSuite RSA:!EXPORT:!DH:!LOW:!NULL:+MEDIUM:+HIGH
```

Comment:
What would be the default for cipher?  Are we going to follow the guidelines in
https://fedoraproject.org/wiki/Packaging:CryptoPolicies ?